asr-9010.yaml

```diff
@@ -0,0 +1,45 @@
+manufacturer: Cisco
+model: ASR-9010
+slug: asr-9010
+part_number: ASR-9010
+u_height: 21
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco ASR 9000 Series Aggregation Services Routers Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/asr-9000-series-aggregation-services-routers/data_sheet_c78-501767.html)'
+power-ports:
+- name: 0/PT0-PM0
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT0-PM1
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT0-PM2
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT0-PM3
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM0
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM1
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM2
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM3
+  type: DC Terminal
+  maximum_draw: 2100
```

Comment:
These should be modelled as removalble modules with the modules included (Ex: A9K-DC-PEM-V2=) in the module definitions